libs/packages/sam-formly/src/lib/formly/wrappers/group.wrapper.ts

```diff
@@ -11,46 +11,51 @@ import * as qs from 'qs';
  */
 @Component({
   template: `
-    <ng-container [ngSwitch]="to.group">
-      <ng-container *ngSwitchCase="'accordion'">
-        <sds-accordion multi="true" displayMode="basic">
-          <sds-accordion-item
-            class="sds-accordion__panel"
-            [expanded]="modelHasValue()"
-          >
-            <sds-accordion-item-header>
-              <span
-                *ngIf="!to.hideLabel"
-                [attr.aria-hidden]="!to.announceLabel ? undefined : 'true'"
-              >
-                {{ to.label }}
-              </span>
-            </sds-accordion-item-header>
-            <ng-container #fieldComponent></ng-container>
-          </sds-accordion-item>
-        </sds-accordion>
-      </ng-container>
-      <ng-container *ngSwitchCase="'panel'">
-        <div
-          class="sds-panel"
-          [ngClass]="{ 'sds-panel--multiple': field?.fieldGroup?.length }"
-        >
+    <ng-container *ngIf="!to.readonlyMode; else defaultTemplate">
```

Comment:
Main point to note here is that group wrappers will be bypassed if readonly mode is enabled in template options

Reply:
👍